src/test/v2/accessibility/accessibility_test.ts

```diff
@@ -0,0 +1,127 @@
+});
+
+
+Scenario('I as an admin should be able to register private beta citizen', async ({ I }) => {
+  const privateBetaRole = await I.haveRole();
+  const privateBetaService = await I.haveService({onboardingRoleNames: [privateBetaRole.name]});
+  const privateBetaAdminRole = await I.haveRole({assignableRoleNames: ['citizen', privateBetaRole.name]});
+
+  const testSecret = faker.internet.password({prefix: '0Ab'});
+  const betaAdmin = await I.haveUser({
+    password: testSecret, 
+    roleNames: [privateBetaAdminRole.name, 'idam-user-dashboard--access']
+  });
+  I.loginAs(betaAdmin.email, testSecret);
+
+  const registerForename = faker.person.firstName();
+  const registerSurname = faker.person.lastName();
+  const registerEmail = faker.internet.email({firstName : registerForename, lastName : registerSurname, provider: 'test.local'});
+  I.navigateToRegisterUser();
+  I.fillField('email', registerEmail);
+  I.click('Continue');
+  I.fillField('#forename', registerForename);
+  I.fillField('#surname', registerSurname);
+  I.click('Private Beta Citizen');
+  I.click('Continue');
+  I.selectOption('#service', privateBetaService.clientId);
+
+  runAccessibilityCheck(I);
+});
+
+Scenario('I as an admin should be able to register support user', async ({ I }) => {
+
+  const registerForename = faker.person.firstName();
+  const registerSurname = faker.person.lastName();
+  const registerEmail = faker.internet.email({firstName : registerForename, lastName : registerSurname, provider: 'test.local'});
+  I.navigateToRegisterUser();
+  I.fillField('email', registerEmail);
+  I.click('Continue');
+  I.fillField('#forename', registerForename);
+  I.fillField('#surname', registerSurname);
+  I.click('Support');
+  I.click('Continue');
+
+  runAccessibilityCheck(I);
+});
+
+Scenario('I as an admin can see errors for invalid search values',  ({ I }) => {
+
+  I.navigateToSearchUser();
+  I.fillField('search', 'email..@test.com');
+  I.click('Search');
+
+  runAccessibilityCheck(I);
+});
+
+Scenario('login as user without access', async ({ I }) => {
+  const testSecret = faker.internet.password({prefix: '0Ab'});
+  const testUser = await I.haveUser({password: testSecret});
+  I.amOnPage('/');
+  I.fillField('Email', testUser.email);
+  I.fillField('Password', secret(testSecret));
+  I.click('Sign in');  
+
+  runAccessibilityCheck(I);
+});
+
+Scenario('I as an admin can suspend user',  async ({ I }) => {
+  const testUser = await I.haveUser();
+  I.navigateToManageUser(testUser.email);
+  I.click('Suspend user');
+  I.checkOption('#confirmSuspendRadio');
+  I.click('Continue');
+
+  runAccessibilityCheck(I);
+});
+
+Scenario('view admin user details',  ({ I }) => {
+  const adminEmail = codeceptjs.container.support('adminIdentity').email;
+  I.navigateToManageUser(adminEmail);
+
+  runAccessibilityCheck(I);
+});
+
+async function runAccessibilityCheck(I: any) {
+  I.runA11yCheck({ outputDir: 'functional-output/accessibility' });
```

Comment:
Do you need both? looking at this example it's not clear if one replaces the other: https://medium.com/bear-plus/accessibility-testing-with-codeceptjs-playwright-and-axe-a72e726b6291